backend/app/views/spree/admin/promotion_code_batches/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<% admin_breadcrumb(link_to plural_resource_name(Spree::Promotion), spree.admin_promotions_path) %>
+<% admin_breadcrumb(link_to @promotion.name, spree.admin_promotion_path(@promotion.id)) %>
+<% admin_breadcrumb(plural_resource_name(Spree::PromotionCodeBatch)) %>
+
+<% content_for :page_actions do %>
+  <li>
+    <% if can?(:create, Spree::PromotionCodeBatch) %>
+      <%= button_link_to Spree.t(:new_promotion_code_batch), new_object_url %>
+    <% end %>
+  </li>
+<% end %>
+
+<% if @promotion_code_batches.any? %>
+  <table>
+    <thead>
+      <tr>
+        <th>Base code</th>
+        <th>Total codes</th>
+        <th>Status</th>
+        <th>Email</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @promotion_code_batches.each do |promotion_code_batch| %>
+        <tr>
+          <td><%= promotion_code_batch.base_code %></td>
+          <td><%= promotion_code_batch.number_of_codes %></td>
+          <td>
+            <%= promotion_code_batch.status %>.
```

Comment:
As Clarke already mentioned, please return a symbol as status and pass the status here to `Spree.t` with a meaningful scope (ie. `promotion_code_batches.status`, so this can be translated in foreign languages.